### DISCUSSION TITLE
My network ignoring devices to join

### DISCUSSION THREAD

**Discussion by bigboban** [Koenkk/zigbee2mqtt#11960] [General]
I have HA + zigbee2mqtt plugin + Slaesh zigbee USB coordinator. Everything was fine for months. But now i had problem with one devices (Ikea Fyrtur). So i tried to re-join but it is not possible. I tried some other devices (Xiaomi button, Ikea button) i used before without problem. but my zigbee networks IGNORES any join try. I have join permitted of course. But after join try (in most cases some long push of hidden button) NO LOG MESSAGE about joining in zigbee2mqtt log.

I am able to remove device from network but i am NOT able to rejoin any way. I restarted zigbee2mqtt, restarted HA, removed Slaesh from USB and put back. Nothing helped.

Please help how to make joining working again.

**Comment by grasgruenit** [comment]
Same problem here with same USB coordinator slaeh's CC2652RB stick.
The network is running fine and joining used to work. Sometimes I had to reboot the RPI to allow joining. Now it does not work at all.
